mailviews/previews.py

```diff
@@ -72,25 +76,35 @@ def register(self, cls):
 
     @property
     def urls(self):
-        urlpatterns = patterns('',
+
+        urlpatterns = [
             url(regex=r'^$',
                 view=self.list_view,
                 name='list'),
             url(regex=r'^(?P<module>.+)/(?P<preview>.+)/$',
                 view=self.detail_view,
                 name='detail'),
-        )
+        ]
+
+        if not is_django_version_greater_than_1_9:
+            urlpatterns = patterns('',
+                urlpatterns
+            )
 
         if not should_use_staticfiles():
-            urlpatterns += patterns('',
+            url_staticsfiles = [
                 url(regex=r'^static/(?P<path>.*)$',
-                    view='django.views.static.serve',
+                    view=django.views.static.serve,
                     kwargs={
                         'document_root': os.path.join(os.path.dirname(__file__), 'static'),
                     },
-                    name='static'),
-                )
-
+                    name='static')
+            ]
+            
+            if is_django_version_greater_than_1_9:
+                urlpatterns += url_staticsfiles
+            else:
+                urlpatterns += patterns('', url_staticsfiles)
```

Comment:
I'm pretty sure that Django 1.5 and up support just using a plain list instead of `patterns` for urls, as long as you don't need the string-handling functionality. So we shouldn't need to automatically wrap the patterns.